charts/hdfs-namenode-k8s/templates/namenode-statefulset.yaml

```diff
@@ -194,6 +200,14 @@ spec:
             name: fs
           - containerPort: {{ .Values.global.nameNodeHttpPort }}
             name: http
+
+          livenessProbe:
+            exec:
+              command:
+              - /nn-scripts/check-active.sh
+            initialDelaySeconds: 60
+            periodSeconds: 600
```

Comment:
The command takes about two seconds, so perhaps `periodSeconds` can be brought down to `300` without impacting performance too much. Should we also perform this check as a readiness probe? In that case `periodSeconds` can probably be substantially lower (e.g., `60`)

Reply:
I set periodSeconds to 200 and failureThreshold to 2